src/view/translations/en.json

```
@@ -49,5 +49,54 @@
     "toolbar-microbit-gpio.tryItDescription": "If you would like to see this sensor supported, please +1 the feature addition issue on GitHub! For now, you can try it on MakeCode!",
     "toolbar-microbit-wireless.title": "Bluetooth  & Radio",
     "toolbar-microbit-wireless.description": "micro:bits can use radio waves and bluetooth services to communicate with each other",
-    "toolbar-microbit-wireless.tryItDescription": "If you would like to see this sensor supported, please +1 the feature addition issue on GitHub! For now, you can try it on MakeCode!"
+    "toolbar-microbit-wireless.tryItDescription": "If you would like to see this sensor supported, please +1 the feature addition issue on GitHub! For now, you can try it on MakeCode!",
+    "toolbar-microbit-gesture-sensor.title": "Gesture",
+    "toolbar-microbit-gesture-sensor.description": "If you move your BBC micro:bit in a certain way (as a gesture) then micro:bit is able to detect this. micro:bit is able to recognise the following gestures: up, down, left, right, face up, face down, freefall, 3g, 6g, 8g, shake.)",
+    "toolbar-microbit-gesture-sensor.tryItDescription": "Select a gesture and send it by clicking the button.",
+    "toolbar-microbit-compass-sensor.title": "Compass",
+    "toolbar-microbit-compass-sensor.description": "A digital compass is an input sensor that detects magnetic fields. Your BBC micro:bit has an inbuilt compass that can detect the direction in which it is facing.",
+    "toolbar-microbit-compass-sensor.tryItDescription": "If you would like to see this sensor supported, please +1 the feature addition issue on GitHub!",
+    "toolbar-clue-temperature-sensor.title": "Temperature",
+    "toolbar-clue-temperature-sensor.description": "CLUE uses the BMP280 sensor, an environmental sensor with temperature.This precision sensor from Bosch is the best low-cost, precision sensing solution for measuring temperature with ±1.0°C accuracy.",
+    "toolbar-clue-temperature-sensor.tryItDescription": "You can set the temperature range from your code!",

```

```suggestion
    "toolbar-clue-temperature-sensor.tryItDescription": "You can simulate the temperature in your code!",

```
Range makes it seem like values from [a, b] when it is just one value.